.github/workflows/ci.yml

```diff
@@ -92,7 +92,7 @@ jobs:
     strategy:
       matrix:
         python: ['3.8', '3.9', '3.10']
-        django: ['3.2', '4.0']
+        django: ['3.2', '4.0', '4.1', '4.2']
```

Comment:
best to drop Django 4.0 from the test matrix as all support ended 6 months ago - https://endoflife.date/django